.changeset/nasty-pets-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-org': patch
+---
+
+Added relations.memberof filter to the catalog api call in MemberListCard to avoid fetching the whole User kind entity from catalog-backend.
```

Comment:
```suggestion
Added `relations.memberof` filter to the catalog api call in `MemberListCard` to avoid fetching all the User entity kinds from catalog-backend.
```

Reply:
if you accept this suggestion in the Github UI make sure that `Signed-off-by: Paladetch Usavasereelert <paladetch.u@gmail.com>` is copied to the commit message description or the DCO bot will complain